Gems/ROS2/Code/Source/RobotImporter/Utils/SourceAssetsStorage.cpp

```diff
@@ -318,26 +322,46 @@ namespace ROS2::Utils
             }
 
             AZ::IO::Path resolvedPath(resolved);
+            const bool needsVisual = visuals.contains(unresolvedUrfFileName);
             const bool needsCollider = colliders.contains(unresolvedUrfFileName);
 
             const AZStd::string pathSuffix = needsCollider ? "_collider" : "";
             const AZStd::string newFilename = resolvedPath.Stem().String() + pathSuffix + resolvedPath.Extension().String();
```

Comment:
Now that the exported model files retain the file suffix (ex: "model.dae" -> "model.dae.azmodel"), is the pathSuffix still needed here?

Reply:
Wtih recent changes to o3de path suffix is not needed anymore. happy to remove this hack.